lib/customer.rb

```diff
@@ -0,0 +1,35 @@
+require "CSV"
+
+class Customer
+  attr_reader :id

```

Comment:
You have `customer.rb` twice in this submission! Please make sure to only submit one copy of each file.

Reply:
This may be related to what was going on with git.